lib/ts/PriorityQueue.h

```diff
@@ -110,6 +110,12 @@ PriorityQueue<T, Comp>::pop()
     return;
   }
 
+  // SKH - I suspect this assignment is not preserving entry indices correctly.
```

Comment:
No need to put your initials in the comment as `git blame` will tell us who added this if someone is curious.